api/constants.py

```diff
@@ -53,3 +53,178 @@
     "commentaire application",
     "commentaire question",
 ]
+
```

Comment:
commentaire plus generic sur les constants : j ai l impression que tu pourrais utiliser les classes `Enum` de python, et ça rendrait sans doute le truc plus simple à utiliser pour toi dans le reste du code.

```diff
- QUESTION_FEEDBACK_LIKE = "like"
- QUESTION_FEEDBACK_DISLIKE = "dislike"
- QUESTION_FEEDBACK_CHOICES = [
-     (QUESTION_FEEDBACK_LIKE, "Positif"),
-     (QUESTION_FEEDBACK_DISLIKE, "Négatif"),
- ]
```
```python
from enum import Enum

QuestionFeedback = Enum(
    value='QuestionFeedback',
    names=[
        ('LIKE', "like"),
        ('DISLIKE', "dislike"),
        # aliases
        ('Positif', "like"),
        ('Négatif', "dislike"),
    ]
)
```
and that's really cool because then you can do
```python
>>> QuestionFeedback.LIKE
<QuestionFeedback.LIKE: 'like'>
>>> QuestionFeedback["Négatif"]
<QuestionFeedback.DISLIKE: 'dislike'>
```
and that improves readability a lot in my opinion

more details https://www.notinventedhere.org/articles/python/how-to-use-strings-as-name-aliases-in-python-enums.html

Reply:
ah je ne connais pas, je vais aller lire ca merci 💯